PowerShell/ModuleChangelog.md

```diff
@@ -1,3 +1,10 @@
+## 1.24.0
```

Comment:
I think my only question here is whether this should be 1.23.4? We technically haven't added any functionality